src/acme/pd/Ticket.java

```diff
@@ -215,18 +253,16 @@ public LocalDateTime getDeliveryTime() {
     }
 
     public void setDeliveryTime(LocalDateTime deliveryTime) {
-//        if (new Random().nextInt() % 3 == 0)
-//            //this.bonus = company.getBonus();
-//        else
-//            this.bonus = new BigDecimal(0);
+        // TODO Bonus
```

Comment:
Calculating bonuses should be covered by this user story.

Reply:
Nope, completion